test/program_test.cpp

```diff
@@ -56,6 +60,38 @@ TEST_CASE(program_default_copy_construct)
     EXPECT(x == y);
 }
 
+TEST_CASE(program_print)
+{
+    migraphx::program p = create_program();
+    auto* mm            = p.get_main_module();
+    auto in1            = mm->end();
+
+    // print end instruction
+    p.debug_print(in1);
```

Comment:
This should use the module.

Reply:
The purpose here is to cover the program::print() function